### ISSUE TITLE
Permission tags may be missing when using separate authentication and authorization backend modules

### ISSUE THREAD

**Issue by rtraschke** [rabbitmq/rabbitmq-server#338]
When configuring separate authentication and authorization backends, for example with the LDAP auth backend,

```
{rabbit, [{auth_backends, [{rabbit_auth_backend_ldap, rabbit_auth_backend_internal}]}]}
```

then any permission tags found when applying the authorisation module fail to get propagated into the user record, only permission tags found during authentication are applied.

The upshot of this can be seen using the above configuration when a user is properly configured to have Rabbit MQ login access via LDAP authentication, but the permission tag for the management UI is only stored in the user record within the Rabbit MQ internal database, such a user will not be granted access to the management UI.

**Comment by michaelklishin** [comment]
Fixed and will be in `3.5.6`: that you, @rtraschke!

**Comment by michaelklishin** [comment]
…and @Gsantomaggio.
